azure-iot-device/azure/iot/device/custom_typing.py

```diff
@@ -48,3 +48,28 @@ class StorageInfo(TypedDict):
     containerName: str
     blobName: str
     sasToken: str
+
+
+class RegistrationState(TypedDict):
+    deviceId: str
+    assignedHub: str
+    subStatus: str
+    createdDateTimeUtc: str
+    lastUpdatedDateTimeUtc: str
+    etag: str
+    payload: JSONSerializable
+
+
+RegistrationPayload = Union[dict, object, None]
```

Comment:
>Union[dict, object, None] [](http://example.com/codeflow?start=22&length=25)

This seems particularly broad -- I think you're saying it could be anything except a string, integer, bool, or float? :)

Reply:
yes..